browser_patches/webkit/build.sh

```diff
@@ -11,22 +11,30 @@ build_gtk() {
   if ! [[ -d ./WebKitBuild/GTK/DependenciesGTK ]]; then
     yes | WEBKIT_JHBUILD=1 WEBKIT_JHBUILD_MODULESET=minimal WEBKIT_OUTPUTDIR=$(pwd)/WebKitBuild/GTK DEBIAN_FRONTEND=noninteractive ./Tools/Scripts/update-webkitgtk-libs
   fi
-  local CMAKE_ARGS=""
+  local CMAKE_ARGS=(
+    --cmakeargs=-DENABLE_INTROSPECTION=OFF
```

Comment:
should it have `--cmakeargs=-DENABLE_WEBXR=OFF` too?

Reply:
Sorry, I should have clarified this.

WebXR is disabled for all WebKit ports except WPE:

https://github.com/WebKit/WebKit/blob/main/Source/cmake/WebKitFeatures.cmake#L236
https://github.com/WebKit/WebKit/blob/main/Source/cmake/OptionsWPE.cmake#L74

The former `--no-webxr` flag in WebKitGTK build options was redundant.